### DISCUSSION TITLE
Roam import fails upon today page collision

### DISCUSSION THREAD

**Discussion by phrohdoh** [athensresearch/athens#924] [Archive]
Reported [on Discord][chae-discord-msg-report] by _chae_:
![image](https://user-images.githubusercontent.com/4861023/114058149-d3a0b200-9858-11eb-88b0-355bdea04718.png)

When this happens, [_nothing_ is imported] into Athens.

### Known Workarounds

- [shared by _Yifan_][yifan-discord-msg-workaround]
  - delete today page in Roam
  - download EDN
  - import into Athens
  - ✅ [confirmed worked][chae-discord-msg-successful-import] for _chae_.

[chae-discord-msg-report]: https://discord.com/channels/708122962422792194/802652338619285594/829740854142173198
[_nothing_ is imported]: https://discord.com/channels/708122962422792194/802652338619285594/829745542967197737
[yifan-discord-msg-workaround]: https://discord.com/channels/708122962422792194/802652338619285594/829744553996320768
[chae-discord-msg-successful-import]: https://discord.com/channels/708122962422792194/802652338619285594/829746629308448779

**Comment by phrohdoh** [comment]
Roam-exported EDN importing implemented in https://github.com/athensresearch/athens/pull/918.

**Comment by pgoodjohn** [comment]
I had this problem too, for me deleting today's page in Athens solved the issue and I was able to import the database.
